prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact Us</title>
+  </head>
+  <body>
+    <h1>Contact Us</h1>
+    <div>
+      <a
+        href="file:///C:/Users/antho/repos/prep-1121-code-solutions/prep-html-links/index.html"
+        >Click here to go to the home page</a
+      >
```

Comment:
Try to use a relative URL for your `href`. Otherwise, your code won't work on anyone else's computer, since this URL has your username in it. For example:

```html
<a href="../index.html">Click here to go to the home page</a>
```

Reply:
i can't figure this out. i was able to update the link to the home page to link the contact us page but i can't seem to update the contact us page to link to the home page anymore
![Screenshot 2021-11-07 150007](https://user-images.githubusercontent.com/93688727/140665117-7d27b222-0950-4256-afc6-179398f5c8d1.jpg)
![Screenshot 2021-11-07 150040](https://user-images.githubusercontent.com/93688727/140665119-f26f23cd-16dc-407f-a50a-751123bd3b25.jpg)